packages/miniflare/src/workers/shared/blob.worker.ts

```diff
@@ -241,6 +243,8 @@ export class BlobStore {
 	}
 
 	async delete(id: BlobId): Promise<void> {
+		// If sticky blobs are enabled, don't delete any blobs
+		if (this.#stickyBlobs) return;
```

Comment:
Do persisted sticky blob "ever" get deleted? Or do we just assume they will be tidied up by the OS at somepoint?